vllm/model_executor/model_loader/weight_utils.py

```diff
@@ -128,6 +128,14 @@ def get_quant_config(model_config: ModelConfig,
     if model_config.quantization == "gguf":
         return quant_cls.from_config({})
 
+    if model_config.quantization == "hqq":
```

Comment:
instead of having an if statement here, could `HQQConfig.from_config` instead take the raw config, such that we can use the standard call in line 151 and avoid special casing here? 

And then we do the unpacking inside `from_config`